### PR TITLE
Better: Add features scope fields on EM sources.

### PR DESCRIPTION
**Pull request by CedricBm**
Add these new fields in the GET and PUT /content_sources

- live_chat
- enable_android
- enable_ios
- enable_web

### CODE REVIEW

File: specs/engage-digital_postman2.json [hunk 1]
```diff
@@ -1672,6 +1737,24 @@
...
+                      "key": "enable_ios",
+                      "value": "\u003cboolean\u003e",
+                      "description": "Only on Engage Messaging sources that are not on livechat mode. Enable or disable the source on iOS.",
+                      "disabled": true
+                    },
+                    {
+                      "key": "enable_web",
+                      "value": "\u003cboolean\u003e",
+                      "description": "Only on Engage Messaging sources that are not on livechat mode. Enable or disable the source on the web.",
+                      "disabled": true
```
Comment (CedricBm) [inline specs/engage-digital_postman2.json:1757]: The real are here.

File: specs/engage-digital_postman2.json [hunk 2]
```diff
@@ -10,6 +10,9 @@
       "item": [
         {
           "name": "Events",
+          "description": {
+            "type": "text/plain"
+          },
```
Comment (CedricBm) [inline specs/engage-digital_postman2.json:15]: hey @CoCraig I followed the instructions in https://github.com/ringcentral/engage-digital-api-docs/tree/master/specs and updated the postman file using swaggman, but somehow I ended up with all these changes. Do you know why? Maybe you updated swaggman?

File: specs/engage-digital_openapi3.yaml [hunk 3]
```diff
@@ -735,6 +735,24 @@ paths:
...
+          name: enable_ios
+          required: false
+          schema:
+            type: boolean
+        - description: Only on Engage Messaging sources that are not on livechat mode. Enable or disable the source on the web.
+          in: query
+          name: enable_web
+          required: false
+          schema:
+            type: boolean
```
Comment (CedricBm) [inline specs/engage-digital_openapi3.yaml:755]: I don't provide the `live_chat` field in the PUT because it is techteam only.

**Comment by CedricBm** [comment]
hey @nel I wanted to assign @brenauj https://github.com/brenauj as a reviewer to this PR but I can't because he isn't part of the RC organization. Can we do something about it please? Either give us access to add collaborators or add our collaborators yourself?

File: specs/engage-digital_openapi3.yaml (hunk 3 above)
Comment (brenauj) [reply specs/engage-digital_openapi3.yaml:755]: Makes sense :+1:

File: specs/engage-digital_postman2.json (hunk 1 above)
Comment (brenauj) [reply specs/engage-digital_postman2.json:1757]: Looks good to me

File: specs/engage-digital_postman2.json (hunk 2 above)
Comment (CoCraig) [reply specs/engage-digital_postman2.json:15]: @CedricBm John recently updated the postman generation and it looks like it is now adding descriptions for each API added to Postman.  Descriptions were probably not there before which is why all these changes are occurring now.

**Pull request merged**